jdaviz/core/template_mixin.py

```diff
@@ -1517,12 +1520,23 @@ def selected_obj(self):
 
     @property
     def selected_subset_state(self):
+        if self.is_multiselect:
+            subset_states = {}
+            for select_subset in self.selected:
+                if select_subset == self.default_text:
+                    continue
+                subset_group = [s for s in self.app.data_collection.subset_groups if
+                                s.label == select_subset][0]
+                subset_states[select_subset] = subset_group.subset_state
+            return subset_states
         subset_group = [s for s in self.app.data_collection.subset_groups if
                         s.label == self.selected][0]
-        return subset_group.subset_state
+        return {self.selected: subset_group.subset_state}
```

Comment:
other multi-select components return single entries for `selected_obj`, et al, when not in multiselect mode.  Is this needed for logic somewhere else?

Reply:
No, I just wanted to be consistent. I can remove that though.